src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -145,6 +145,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('registry_name', options_list=['--registry', '-r'])
         c.argument('image_name', options_list=['--image', '-t'], help="The name and tag of the image using the format: '-t repo/image:tag'.")
         c.argument('builder', options_list=['--builder', '-b'], help="The name and tag of a Buildpack builder image.")
+        c.argument('pack_image_tag', options_list=['--pack-image-tag'], help="The tag of the 'pack' runner image ('mcr.microsoft.com/oryx/pack').", is_preview=True)
```

Comment:
Can you call out the default is 'stable'?

Reply:
it should mention that when a customer passes `--help`.

Reply:
Is this conversation resolved?